### DISCUSSION TITLE
Could this be a useful purpose for ordinal inscriptions?

### DISCUSSION THREAD

**Discussion by hannpet** [mycitadel/mycitadel-desktop#59] [Ideas]
Suppose this scenario (sorry if I get something very wrong, I'm learning):
A bitcoiner creates a Multisig-wallet, with m bip39 seeds stored at different locations.

Problem: The wallet-descriptor also needs to be stored and never be lost (and idealy kept very private).

The question arises... how and where to store the wallet-descriptor?!
Could ordinal inscriptions be maybe a useful solution for this?

Very naive proposal for storage of the wallet-descriptor:

for each bip39-Seed (as i) do:
  encrypt wallet-descriptor with i, take this text-snippet and inscribe it as ordinal in a standard-wallet of the seed i.

Further... what if this was even integrated in citadel-desktop?
Any of the bip39-seeds for the multisig could be used as "Import-Key", which would automativally retrieve and decrypt the multisig-wallet-descriptor. Then business as usual.

Just a note: I'm no big fan of ordinals, I just like the property that they are on-chain-data that cannot vanish as long as bitcoin exists. For everything else I suppose RGB is the solution (but I admit I am not knowledgeable, RGB seems super awesome though).

I would be interested, if such a "wallet-descriptor-storage" for multisig would make sense, from a security/systematic point of view?

**Comment by dr-orlovsky** [comment]
> they are on-chain-data that cannot vanish as long as bitcoin exists

Nope. Zero-synch would change that: https://www.coindesk.com/tech/2023/03/28/newly-formed-zerosync-association-brings-zero-knowledge-proofs-to-bitcoin/

I am strong proponent of not believing into a "free lunch". The proper way to backup is to pay for the backups, otherwise nobody can guarantee anything. That's why we have Storm channel proposal https://github.com/Storm-WG/storm-spec, as well as Storm data network for LN with [Strom node](https://github.com/Storm-WG/storm-node) and [LNP node](https://github.com/LNP-WG/lnp-node) (for the video demo of how to use it we need to ask @UkolovaOlga who's going to upload it to our [YouTube channel](https://youtube.com/LNP-BP))

**Comment by hannpet** [comment]
Thanks for the info & feedback! Yes, I agree.. with Zero-Sync nobody would download the whole timechain and then it's questionable if the ordinal inscriptions have any lasting persistance in such scenario.

I really like the storm-network concept & look forward for the demo video to hopefully show up in the youtube-channel on how to use it.
Fantastic work! 🥇

**Comment by UkolovaOlga** [comment]
@hannpet you can check the video with Storm demo and explanations here https://youtu.be/jI4QprrHC3w
